### PR TITLE
Updated import placement, print statement, bare except & long line changes

### PR DESCRIPTION
**Pull request by kdcis**
Updated import placement
Updated print statement brackets
Ignored bare except warnings
Updated long line changes

### CODE REVIEW

File: dev/msgtest.py [hunk 1]
```diff
@@ -3,25 +3,25 @@
 from pprint import pprint
 import sys
 import traceback
+import messagetypes
```
Comment (PeterSurda) [inline dev/msgtest.py:6]: Make this into a unit test, maybe add a couple more test cases. This file can be removed completely after the tests are there.

File: dev/powinterrupttest.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 def signal_handler(signal, frame):
     global shutdown
-    print "Got signal %i in %s/%s" % (signal, current_process().name, current_thread().name)
+    print("Got signal %i in %s/%s" % (signal, current_process().name, current_thread().name))
```
Comment (PeterSurda) [inline dev/powinterrupttest.py:14]: This can also be made into a unit test, but in a separate PR.

**Review by PeterSurda — CHANGES_REQUESTED** [review]
make `msgtest.py` into a unit test, otherwise ok.

**Comment by g1itch** [comment]
@PeterSurda is this dir included into the lint checks? As for me, it shouldn't be. I agree that it should be converted to a set of full-fledged unit tests. I tried to do that with `bloomfiltertest` a couple of years ago and came into a conclusion that there is no suitable implementation )`: https://github.com/g1itch/PyBitmessage/tree/tests-bloom

**Comment by PeterSurda** [comment]
@g1itch It isn't included in the checks. I gave instructions to later rewrite these as proper unit tests.

**Comment by g1itch** [comment]
> It isn't included in the checks.

Than the only reason to touch this is inability to find a piece of work, I guess.

File: src/tests/test_msg.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+from pybitmessage import messagetypes
+
+sample_data = {"": "message", "subject": "subject", "body": "body"}
+invalid_data = {"": "message", "subject": b"\x01\x02\x03", "body": b"\x01\x02\x03\x04"}
+
+
+class TestMessageTypes(unittest.TestCase):
+    """A test case for messagetypes"""
+
+    @classmethod
```
Comment (PeterSurda) [inline src/tests/test_msg.py:15]: This can be removed as it just calls parent.

File: src/messagetypes/message.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import logging
 
-from messagetypes import MsgBase
+from pybitmessage.messagetypes import MsgBase
```
Comment (PeterSurda) [inline src/messagetypes/message.py:3]: This needs to be tested outside of installed environment.

Comment (PeterSurda) [reply src/messagetypes/message.py:3]: Unit test works, but could use more testing.

**Review by PeterSurda — CHANGES_REQUESTED** [review]
Make the requestd changes

File: src/messagetypes/message.py (hunk 4 above)
Comment (kdcis) [reply src/messagetypes/message.py:3]: Tested & working good.

**Review by PeterSurda — APPROVED** [review]

**Review by PeterSurda — CHANGES_REQUESTED** [review]
This should be split into 2 PRs.

One is the `print` fixes and removing obsolete files.
One is the messagetypes related stuff

Alternatively, if it's easier, you can split it into 2 commits instead of 2 PRs.

**Comment by kdcis** [comment]
> This should be split into 2 PRs.
> 
> One is the `print` fixes and removing obsolete files. One is the messagetypes related stuff
> 
> Alternatively, if it's easier, you can split it into 2 commits instead of 2 PRs.

Updated commits accordingly, Please check

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**